### DISCUSSION TITLE
Github account Suspended

### DISCUSSION THREAD

**Discussion by moviehall** [community/community#138884] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Please sir release my account from suspension it's been more than 1 month. I have already fill the account reinstatement form. But I'm still waiting. I didn't get any mail for what reason my account got Suspended. I think this is because of one of my repositorie. Then I'm ready to delete it. Why it's taking so much time when i have not done anything wrong. I'm facing many difficulties after account suspension.

MY github account username is ScripterSaurav

Please someone help me.

**Comment by jovanovic-djo** [comment]
Contact Github support: https://support.github.com/ , and they will eventually activate your account. 
Do not worry, it would take only a few days.

There are plenty of reddit posts about similar occurrences, you can read them for more info: https://www.reddit.com/r/github/comments/jenp4h/my_github_account_suddenly_got_suspended/
https://www.reddit.com/r/github/comments/14g7xvq/github_account_suspended/
https://www.reddit.com/r/github/comments/mtnzxj/github_account_suspended_i_havent_even_used_it_yet/
https://www.reddit.com/r/github/comments/18yelze/github_account_suspended_and_no_response_from/

**Comment by ebndev** [comment]
Hi @moviehall,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
